charts/eks-anywhere-packages/templates/deployment.yaml

```diff
@@ -90,11 +95,18 @@ spec:
           - mountPath: /tmp/k8s-webhook-server/serving-certs
             name: cert
             readOnly: true
+          - mountPath: /tmp/registry-mirror
+            name: cert
+            readOnly: true
       volumes:
       - name: cert
         secret:
           defaultMode: 420
           secretName: webhook-server-cert
```

Comment:
Seems like the mode should be 0200 for these things, but not a huge deal. Also odd to me to use the /tmp directory, but that is what the webhooks are doing. It does seem like a problem that the volume mount name is the same for webhooks and the registsry.